client/webserver/site/src/html/register.tmpl

```diff
@@ -39,11 +39,6 @@
       {{template "newWalletForm" "[[[reg_dcr_required]]]"}}
     </form>
```

Comment:
Seems like the `WalletStep` form is always hidden now too and can be removed as well.

Reply:
Actually this form stills in use, when setting the wallet.